webapp/context.py

```diff
@@ -0,0 +1,5 @@
+import datetime
+
+
+def current_year():
```

Comment:
I'm sure you copied this `webapp/context.py` pattern from somewhere, but for me this is really a terrible reason for creating a whole new function, let alone a file.

Could you just do:

``` python3
@app.context_processor
def context():
    return {"current_year": datetime.datetime.now().year}
```